### ISSUE TITLE
Baseline GNN Class design

### ISSUE THREAD

**Issue by sebastian9991** [ekmpa/CrediGraph#34]
I'm trying to understand which architecture is most reasonable for our purposes. 

https://pytorch-geometric.readthedocs.io/en/latest/cheatsheet/gnn_cheatsheet.html

**Comment by sebastian9991** [comment]
Perhaps we can start a simple sym-normalized aggregation and self-loop. That is, a GCN for a simple baseline

**Comment by sebastian9991** [comment]
Here are my thoughts: 

- Basic GNN as a baseline. 

- A GCN architecture, seeing if normalization and self-loop lead to better performance, numerical stability and less overfitting. 

- A GAT architecture, to experiment whether attention-based GNN would learn to ignore/capture certain degree patterns in message passing relevant to credibility assignment.
